src/html/faq.html

```diff
@@ -12,7 +12,7 @@ <h1 class="text text--title-1">FAQ</h1>
         </div>
         <div class="page-title__text-2">
           <p class="text text--body-2 text--black">
-            Need additional information? Contact us at nicholasborror@aol.com.
+            Need additional information? Contact us at partypartydsm@gmail.com.
```

Comment:
`<a href="mailto:partypartydsm@gmail.com">partypartydsm@gmail.com</a>` would make a link that, when clicked, would open the user's email client. This isn't super helpful on desktop when folks like me use something like GMail in the browser because it won't open that, but it could be on mobile. Totally optional - up to you - just figured I'd bring it up in case you didn't know it was a thing.

Reply:
Oh, awesome. I definitely didn't know it was a thing. Good call - thank you!